src/main/java/com/laserfiche/repository/api/clients/impl/EntriesClientImpl.java

```diff
@@ -146,7 +143,9 @@ public CreateEntryResult importDocument(ParametersForImportDocument parameters)
                             httpRequestHandler);
                     httpResponse = httpClient
                             .post(requestUrl)
-                            .field("electronicDocument", parameters.getInputStream(), parameters.getFileName())
+                            .field("electronicDocument", parameters.getInputStream(),
+                                    parameters.getContentType() == null ? ContentType.APPLICATION_OCTET_STREAM : ContentType.create(
```

Comment:
previously, no content type was set on the `electronicDocument` parameter. unirest will default to `ContentType.APPLICATION_OCTET_STREAM` in this case. Since we allow setting content type now, if content type is null, we will set it to the default `ContentType.APPLICATION_OCTET_STREAM` to keep similar behaviour as before.